specs/phase1/beacon-chain.md

```diff
@@ -425,7 +425,7 @@ def compute_previous_slot(slot: Slot) -> Slot:
 #### `pack_compact_validator`
 
 ```python
-def pack_compact_validator(index: ValidatorIndex, slashed: bool, balance_in_increments: uint64) -> uint64:
+def pack_compact_validator(index: ValidatorIndex, slashed: boolean, balance_in_increments: uint64) -> uint64:
```

Comment:
This should be a `bool`, but the below formula that shifts the boolean value is unafe (although python converts it to `int` automatically). It should convert the `bool` into a `uint64`, then shift it afterwards. Good find.

Reply:
Thanx! Corrected